Fitness/Controllers/HomeController.swift

```diff
@@ -226,8 +246,11 @@ extension HomeController: UICollectionViewDelegate, UICollectionViewDelegateFlow
     }
 
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-        if collectionView != mainCollectionView {
-            return CGSize(width: 228.0, height: 195.0)}
+        if collectionView.accessibilityIdentifier == Identifiers.todaysClassesCell{
+            return CGSize(width: 228.0, height: 195.0)
```

Comment:
These constants should be brought out somewhere for named context. Either at the top of the function or if reused multiple places in a caseless enum at the top of the file.